### DISCUSSION TITLE
Disk Images and Application Bundles

### DISCUSSION THREAD

**Discussion by probonopd** [helloSystem/hello#272] [Show and tell]
Please check out this helloSystem Development Update: Disk Images and Application Bundles.

What do you think about it?

https://www.youtube.com/watch?v=jQsPPZultWo

**Comment by grahamperrin** [comment]
Whilst https://github.com/helloSystem/hello/issues/203 does have some potential to make helloSystem less prone to breakage, it is, holistically, probably not the best approach. 

Better: https://github.com/helloSystem/ISO/issues/51
